src/docfx/build/page/Transformer.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class Transformer
```

Comment:
>Transformer [](start = 26, length = 11)

Too general, XrefTransformer?

Reply:
It is not only used for Xref, it is quite general

Reply:
AtributeTransformer

---
In reply to: [228430628](https://github.com/dotnet/docfx/pull/3531#discussion_r228430628) [](ancestors = 228430628)